pandas/core/indexes/datetimes.py

```diff
@@ -663,14 +663,14 @@ def _get_time_micros(self):
             values = self._data._local_timestamps()
         return fields.get_time_micros(values)
 
-    def to_series(self, keep_tz=None, index=None, name=None):
+    def to_series(self, keep_tz=lib._no_default, index=None, name=None):
```

Comment:
FYI we should standard this argument somewhere and actually have a check that we don't have multiple ways of doing this (a maintenance concern).